src/garage/torch/policies/continuous_mlp_policy.py

```diff
@@ -0,0 +1,70 @@
+"""
+This modules creates a continuous MLP policy network.
+
+A continuous MLP network can be used as policy method in different RL
+algorithms. It accepts an observation of the environment and predicts an
+action.
+"""
+import akro
+import torch
+
+from garage.misc.overrides import overrides
+from garage.tf.policies.base import Policy
+
+
+class ContinuousMLPPolicy(Policy):
+    """
+    This class implements a policy network.
+
+    The policy network selects action based on the state of the environment.
+    It uses neural nets to fit the function of pi(s).
+    """
+
+    def __init__(self,
+                 env_spec,
+                 nn_module,
+                 name='ContinuousMLPPolicy',
+                 input_include_goal=False):
+        """
+        Initialize class with multiple attributes.
+
+        Args:
+            env_spec():
```

Comment:
Please delete lines of documentations which don't add anything (in this case, the `env_spec():` and `nn_module():` lines.